project_national/fact_sheets_category_10.yml

```diff
@@ -0,0 +1,1190 @@
+
+    reporting_measures:
+      - measure_dir_name: QOIReport
+
+    server_directory_cleanup:
+      retain_in_idf: false
+      retain_schedules_csv: false
+
+
+### -- Upgrades ---
+# Category 10:  Heat Pumps with Existing Backup + Light Touch Envelope
+upgrades:
+  - upgrade_name: EnergyStar ducted ASHP w/ existing natural gas backup + light touch envelope
+    # Category 6, Measure 1 + Category 2, Measure 1
+
+    package_apply_logic: &dualfuel_ASHP_apply_logic_NG
+      - HVAC Has Ducts|Yes
+      - Heating Fuel|Natural Gas
+      - not:
+          or:
+            - HVAC Heating Efficiency|Shared Heating
+            - HVAC Cooling Efficiency|Shared Cooling  
+
+    options:
+      #ASHP
+      - &ducted_ASHP_EnergyStar_exist_backup_no_shared_ducts_NG
+        option: HVAC Heating Efficiency|ASHP, SEER 16, 9.2 HSPF, Separate Backup
+        apply_logic: &dualfuel_ASHP_no_ducted_furnace_apply_logic_NG
+          - not: HVAC Heating Type And Fuel|Natural Gas Fuel Furnace
+        costs: &costs_ashp_ducted_SEER_16_9-2_HSPF
+        # Source: Custom regression by Brennan Less. Ducted heat pump project costs were regressed on nameplate tons and HSPF,
+        # from data described in in Less et al. https://doi.org/10.20357/B7FP4D. January 2019 $ inflated to April 2023 $ using 1.21 factor.
+        # Ducted ASHP regression n=317. Implicitly includes electrical upgrades and electric backup.
+          - value: 4821.83 # for HSPF 9.2
+            multiplier: Fixed (1)
+          - value: 167.64
+            multiplier: Size, Heating System Primary (kBtu/h)
+        lifetime: &lifetime_ashp_ducted_SEER_16_9-2_HSPF 15
+
+      # Boilers as secondary systems
+      # Set secondary boiler fuel type
+      - &ng_boiler_secondary_heating_type_fuel
+        option: HVAC Secondary Heating Type And Fuel|Natural Gas Boiler
+        apply_logic:
+          - HVAC Heating Type And Fuel|Natural Gas Fuel Boiler
+      # Set secondary fuel boiler efficiency levels
+      - &secondary_boiler_76
+        option: HVAC Secondary Heating Efficiency|Fuel Boiler, 76% AFUE
+        apply_logic:
+          - HVAC Heating Efficiency|Fuel Boiler, 76% AFUE
+      - &secondary_boiler_80
+        option: HVAC Secondary Heating Efficiency|Fuel Boiler, 80% AFUE
+        apply_logic:
+          - HVAC Heating Efficiency|Fuel Boiler, 80% AFUE
+      - &secondary_boiler_90
+        option: HVAC Secondary Heating Efficiency|Fuel Boiler, 90% AFUE
+        apply_logic:
+          - HVAC Heating Efficiency|Fuel Boiler, 90% AFUE
+
+
+      # Wall/Floor Furnace as secondary systems
+      # Set secondary Wall/Floor Furnace fuel type
+      - &ng_wallfloor_furnace_secondary_heating_type_fuel
+        option: HVAC Secondary Heating Type And Fuel|Natural Gas Fuel Wall/Floor Furnace
+        apply_logic:
+          - HVAC Heating Type And Fuel|Natural Gas Fuel Wall/Floor Furnace
+
+      # Set secondary fuel Wall/Floor Furnace efficiency levels
+      - &secondary_wallfloor_furnace_60
```

Comment:
Need to change all `HVAC Heating Efficiency` for gas to `HVAC Secondary Heating Efficiency`